shell/platform/tizen/tizen_renderer_evas_gl.cc

```diff
@@ -663,7 +663,11 @@ Evas_Object* TizenRendererEvasGL::SetupEvasWindow(int32_t* width,
     efl_util_set_notification_window_level(evas_window_,
                                            EFL_UTIL_NOTIFICATION_LEVEL_TOP);
   }
+  // Please uncomment below and enable setWindowGeometry of window channel when
+  // Tizen 5.5 or later was chosen as default.
+  // elm_win_aux_hint_add(evas_window_, "wm.policy.win.user.geometry", "1");
```

Comment:
Is it only necessary for 5.5 and higher? I thought it was absolutely necessary

Reply:
`elm_win_aux_hint_add` is supported in Tizen 5.5 and later and Our symbol checker filters elm_win_aux_hint_add in Tizen 4.0.

So @swift-kim and I decided to comment it out and to keep that `setWindowGeometry` disabled on evas gl until Tizen 5.5 or later becomes our default.

Reply:
So `elm_win_aux_hint_add(evas_window_, "wm.policy.win.user.geometry", "1")` is required to configure window size?

Reply:
@HakkyuKim  Yes, sorry for the late reply, I was on vacation.